ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/PrivJsonRpcMethods.java

```diff
@@ -106,4 +104,18 @@ protected RpcApi getApiGroup() {
     }
     return RPC_METHODS;
   }
+
+  private JsonRpcMethod distributeRawTransactionMethod(
+      final PrivacyController privacyController,
+      final EnclavePublicKeyProvider enclavePublicKeyProvider) {
+    if (getPrivacyParameters().getGoQuorumPrivacyParameters().isPresent()) {
```

Comment:
That should be in the GoQuorumJsonRpsPrivacyMethods

Reply:
I changed this but now you get unexpected behaviour if you have PRIV and GOQUORUM APIs enabled